meinberlin/assets/scss/components/_timeline-carousel.scss

```diff
@@ -45,7 +45,6 @@ $timeline-carousel__top-space: 11px;
 
 //slick overwrites
 .slick-slide {
-    margin: 0 3*$em-spacer;
```

Comment:
we use this in the project timeline so we can't remove it and not add it back, one way to do it would be to nest these styles within a timeline parent class, I think .timeline_carousel would work

Reply:
hmm..that is strange, let me doublecheck